source/includes/quick-start/overview.rst

```diff
@@ -0,0 +1,14 @@
+This guide shows you how to create an application that uses the **Java driver**
+to connect to a **MongoDB Atlas cluster**. If you prefer to connect to MongoDB
+using a different driver or programming language, see our
+:driver:`list of official MongoDB drivers <>`.
+
+The Java driver lets you connect to and communicate with MongoDB instances
```

Comment:
Nit: is `instances` the word we use in general?  I would think `cluster` or `server`.

Reply:
I don't think we've decided on a term across docs teams and we refer to them in several different ways. I can use cluster across docs-java if you'd prefer.

In docs-node, we used "instances" much more often than cluster, server, or replica set.

Reply:
Let me know what you think of this change from "instance(s)" to "cluster" https://github.com/mongodb/docs-java/pull/3/commits/3693d5e28b8322bbab51e11da2db4d20cd4f64c6